tests/unit/traditional.js

```diff
@@ -30,5 +32,7 @@ exports.testInitializationMultiple = DomUtil.createDomTest(
         test.deepEqual($input.selectivity('value'), [3, 4]);
 
         test.deepEqual($('#selectivity-input').selectivity('value'), [3, 4]);
+
+        test.equal($input.find('select[name="my_select"] option[selected]').length, 2);
```

Comment:
This is a good start. To be complete I think you want to test the values of the `<option>` elements.